docs/how_to/run_session.rst

```diff
@@ -1,57 +1,137 @@
+
+There are two ways to specify a backend in a session:
+
+**Directly specify a string with the backend name.** Example: 
+ 
+  .. code-block:: python
 
-**Example:**
+    backend = "ibmq_qasm_simulator"
+    with Session(service=service, backend=backend):
+      ...
+
+**Pass the backend object.** Example: 
+
+  .. code-block:: python
+
+    backend = service.get_backend("ibmq_qasm_simulator")
+    with Session(service=service, backend=backend):
+      ...
+
+
+Specify the session length
+--------------------------
+
+When a session is started, it is assigned a maximum session timeout value. After the session has been open the specified amount of time, the session expires and is forcefully closed. You can no longer submit jobs to that session.  See `What happens when a session ends <../sessions.html#ends>`__ for further details.
+
+You can configure the maximum session timeout value through the `max_time` parameter, which can be specified as seconds (int) or a string, like "2h 30m 40s".  This value has to be greater then the `max_execution_time` of the job and less than the system’s `max_time`. The default value is the system’s `max_time`. See `What is the maximum execution time for a Qiskit Runtime job? <faqs/max_execution_time.html>`__ to determine the system's `max_time`.
+
+When setting the session length, consider how long each job within the session might take. For example, if you run five jobs within a session and each job is estimated to be five minutes long, the maximum time for the session should at least 25 min. 
 
 .. code-block:: python
 
-  with Session(service=service, backend="ibmq_qasm_simulator"):
-      estimator = Estimator(options=options)
-    
-.. note::
-  When running in IBM Cloud, if you don't specify a backend, the least busy backend is used. 
+  with Session(service=service, backend=backend, max_time="25m"):
+    ...   
+
+There is also an interactive timeout value (5 minutes), which is not configurable.  If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see `How long a session stays active <../sessions.html#active>`__. 
+
+.. _close session:
+  
+Close a session
+---------------
+
+When jobs are all done, it is recommended that you use `session.close()` to close the session. This allows the scheduler to run the next job without waiting for the session timeout,  therefore making it easier for everyone.  You cannot submit jobs to a closed session.  
+
+.. warning::  
+  Close a session only after all session jobs **complete**, rather than immediately after they have all been submitted. Jobs that are not yet queued are converted to fair-share and will likely time out.  Jobs that are queued but not completed will fail.  
```

Comment:
> Jobs that are not yet queued are converted to fair-share and will likely time out.

Jobs submitted to a closed session will be rejected.

Reply:
Yeah, rejecting means your request doesn't even go through (i.e. you don't get a job back), and failing means your request is accepted, you get a job back, but then it fails. What you have now is fine.